.buildkite/pipeline.yml

```diff
@@ -566,6 +566,7 @@ steps:
       - label: "end to end test"
         command: julia --project=calibration/test calibration/test/e2e_test.jl
         artifact_paths: "calibration_end_to_end_test/*"
+        soft_fail: true
```

Comment:
Is this being added due to the timeouts?

Reply:
Yes. @nefrathenrici suggested there are some problems with slurm daemon and its best to set it to soft_fail for now!

Reply:
Yes, IMSS have said the slurm database daemon is down intermittently because it is hosted on  the new storage system.
 ClimaCalibrate uses `sacct` to query jobs and has been getting this error:
```
sacct: error: slurm_persist_conn_open_without_init: failed to open persistent connection to host:head1:6819: Connection refused
sacct: error: Sending PersistInit msg: Connection refused
sacct: error: Problem talking to the database: Connection refused
```

the `build_history` script in buildkite also failed because of this https://buildkite.com/clima/climacoupler-ci/builds/4928#01929cd1-788d-40b6-bbcc-bf0972b32acf/202-210